Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-NewServiceFabricApplication.ps1

```diff
@@ -161,6 +161,11 @@
        $ApplicationName = Get-ApplicationNameFromApplicationParameterFile $ApplicationParameterFilePath
     }
 
+    if($ApplicationName -eq '')
```

Comment:
@NCarlsonMSFT: this change diverges task's copy of this file from original SDK files. Any thought about this divergence?  This could be a problem if say in future we need to sync files from SDK to get new functionalities.

Reply:
They are already diverged, so this won't introduce a new problem in that regard.